msgraphforentra_connector.py

```diff
@@ -0,0 +1,1631 @@
+# File: msgraphforentra_connector.py
+#
+# Copyright (c) 2022-2024 Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+#
+#
+
+import phantom.app as phantom
+from phantom.action_result import ActionResult
+from phantom.base_connector import BaseConnector
+
+try:
+    from urllib.parse import quote, urlencode
+except Exception:
+    from urllib import quote, urlencode
```

Comment:
This is a py2/py3 compat layer thing which is not necessary
```suggestion
from urllib.parse import quote, urlencode
```

Reply:
agreed, hangover from previous apps